pkgs/applications/version-management/git-and-tools/git-extras/default.nix

```diff
@@ -1,16 +1,22 @@
-{ stdenv, fetchurl }:
+{ stdenv, fetchurl, unixtools, which }:
 
 stdenv.mkDerivation rec {
   pname = "git-extras";
-  version = "4.7.0";
+  version = "5.0.0";
 
   src = fetchurl {
     url = "https://github.com/tj/git-extras/archive/${version}.tar.gz";
-    sha256 = "0pab4f5kmmcn333aswkgndf1fgilc41h8h0rk3lviz0yi8j59vaq";
+    sha256 = "16k0zwx5njmmbzkqkyl1xrsx2ykb0j35ygz39wc9s48j9kqhmdvz";
   };
 
+  nativeBuildInputs = [ unixtools.column which ];
+
   dontBuild = true;
 
+  preInstall = ''
+    patchShebangs .
```

Comment:
Doesn't `patchShebangs` get run automatically? It seems to be part of the default `fixupOutputHooks` for everything using `stdenv.mkDerivation`.

Reply:
without the explicit `patchShebangs` the package doesn't event build.